speakers.json

```diff
@@ -1,9 +1,9 @@
 [
   {
-    "name": "You?",
-    "uri": "https:\/\/twitter.com\/berlinjs",
-    "title": "Your talk title",
-    "description": "This is your spot to take! We want to hear your story. Get in touch with us and we're happy to host you!"
+    "name": "Andrey Sitnik",
+    "uri": "https://twitter.com/andreysitnik",
```

Comment:
not sure why they escape in plain text, but you should probably keep the backslashes)